src/gwt/src/org/rstudio/studio/client/workbench/views/console/Console.java

```diff
@@ -55,6 +58,9 @@
       IsWidget getConsoleInterruptButton();
       IsWidget getConsoleClearButton();
       IsWidget getProfilerInterruptButton();
+      IsWidget getSuspendBlockedIcon();
+      IsWidget setSuspendBlockedIcon(SessionSuspendBlockedEvent event);
+      IsWidget getSuspendedIcon();
```

Comment:
```suggestion
      void onSuspendedBlockedEvent(SessionSuspendBlockedEvent event);
      void onSerializationEvent(SessionSerializationEvent event);
```
I suggest replacing these specific methods with more general ones that feed the event/event type through to the view class, so that the view class can handle the visibility logic internally. This way you don't have to "export" these elements out from the view class, and if you want to add more logic/elements, you can modify just the view class without worrying about the model.